config/api.js

```diff
@@ -1,5 +1,5 @@
-// import { API } from "@quantfive/js-web-config";
-import { API } from "./api/index";
+import { API } from "@quantfive/js-web-config";
```

Comment:
Probably a merge thing but I see you removed a critical line. This will cause token issues. Can you update to use `import { API } from "./api/index";` ?